.github/workflows/devbox-update.yml

```diff
@@ -0,0 +1,48 @@
+name: Weekly Devbox Update
```

Comment:
@josvazg would you please take a look here?

Reply:
LGTM, were you able to test it?

Reply:
yes. https://github.com/filipcirtog/mongodb-atlas-kubernetes/pull/51